doc/articles/uno-development/working-with-uno-wpf-host.md

```diff
@@ -1,10 +1,10 @@
 # The Uno.UI WebAssembly WPF Host
```

Comment:
Thanks for your contribution! 

It's actually very outdated documentation, for which the package is no longer supported. We can remove the whole file.

Is this a scenario that you tried to rely on ?